contrib/k8s-lbry/README.md

```diff
@@ -0,0 +1,862 @@
+    + [Startup chainquery with a database snapshot](#startup-chainquery-with-a-database-snapshot)
+  * [lbrynet API service (not for spee.ch)](#lbrynet-api-service-not-for-speech)
+    + [IMPORTANT - Backup your cluster wallet](#important---backup-your-cluster-wallet)
+  * [spee.ch (and lbrynet sidecar and mysql)](#speech-and-lbrynet-sidecar-and-mysql)
+    + [IMPORTANT - Backup your speech wallet](#important---backup-your-speech-wallet)
+    + [Fund your speech wallet](#fund-your-speech-wallet)
+    + [Create a thumbnail channel](#create-a-thumbnail-channel)
+    + [Finish speech setup](#finish-speech-setup)
+- [Extra commands that k8s-lbry (run.sh) provides](#extra-commands-that-k8s-lbry-runsh-provides)
+  * [k8s-lbry helm](#k8s-lbry-helm)
+  * [k8s-lbry kubectl](#k8s-lbry-kubectl)
+  * [k8s-lbry logs](#k8s-lbry-logs)
+  * [k8s-lbry shell](#k8s-lbry-shell)
+  * [k8s-lbry shell-pvc](#k8s-lbry-shell-pvc)
+  * [k8s-lbry restart](#k8s-lbry-restart)
+  * [k8s-lbry lbrynet](#k8s-lbry-lbrynet)
+  * [k8s-lbry chainquery-mysql-client](#k8s-lbry-chainquery-mysql-client)
+  * [k8s-lbry speech-mysql-client](#k8s-lbry-speech-mysql-client)
+  * [k8s-lbry lbrynet-copy-wallet](#k8s-lbry-lbrynet-copy-wallet-)
+  * [k8s-lbry package](#k8s-lbry-package)
+- [TLS / SSL / HTTPS](#tls--ssl--https)
+- [Cloud specific notes](#cloud-specific-notes)
+  * [AWS](#aws)
+  * [minikube](#minikube)
+- [Uninstall](#uninstall)
+
+<!-- tocstop -->
+
+## Requirements
+
+ * A Kubernetes cluster.
+   * Tested on DigitalOcean managed Kubernetes cluster on nodes with 8GB of RAM,
+     on kubernetes 1.14.1.
+   * Tested on AWS with [Charmed Kubenetes
+     Distribution](https://www.ubuntu.com/kubernetes/docs/quickstart) - See
+     [AWS specific notes](#aws).
+   * Tested on
+     [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for a
+     self-contained virtual machine running kubernetes in VirtualBox - - See
+     [minikube specific notes](#minikube).
+ * Local development machine dependencies:
+   * [GNU Bash](https://www.gnu.org/software/bash/) and friends. If you're on
+     Linux or Mac, you should be good to go.
+   * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+      * Tested with kubectl v1.14.0
+   * [helm](https://github.com/helm/helm/releases)
+      * Tested with helm v2.13.1
+ * Optional: for TLS / HTTPs support, you will also need an internet domain
+   name, and the ability to update its DNS.
+
+Your cloud provider should have instructions for setting up `kubectl` to talk to
+your cluster. This usually involves downloading a config file and putting it in
+`$HOME/.kube/config`. (The file has to be renamed `config` and put in the
+`$HOME/.kube` directory.)
+
+Test that your `kubectl` can talk to your cluster, by querying for a list of
+running nodes:
+
+```
+kubectl get nodes
+```
+
+If everything is working, you should see a list of one or more nodes running and
+showing `STATUS=Ready`
+
+## Security Notice
```

Comment:
We need to also refer to vulnerability reporting and any responsible disclosure stuff in a security notice.

Reply:
added!